Makefile

```diff
@@ -284,7 +284,7 @@ $(addprefix generate-submodules-,$(GO_SUB_MODULES)):
 .PHONY: docker-build
```

Comment:
I wonder if we should also set `IMAGE_BUILD_EXTRA_FLAGS` to include this ulimit thing, for folks using the `image-hive` target. [ref](https://github.com/openshift/hive/blob/cfcd32cb8ae7a077ff8cd4244b3f0a4bc7dce4ab/vendor/github.com/openshift/build-machinery-go/make/targets/openshift/images.mk#L22)

(NB: I'm not sure whether that would actually work via imagebuilder -- would need to be tested.)

Reply:
Should be easy enough to make it additive, no? But even if not, I've confirmed there is nothing downstream of our invocation. (And if someone adds something later, it would be a bug on them not to make _theirs_ additive.)